src/main/java/com/commercetools/sync/categories/helpers/CategoryReferenceResolver.java

```diff
@@ -59,8 +59,8 @@ public CategoryReferenceResolver(@Nonnull final CategorySyncOptions options,
      * taken from the id field of the references.
      *
      * @param categoryDraft the categoryDraft to resolve it's references.
-     * @return a {@link CompletionStage} that contains as a result a new categoryDraft instance with resolved category
-     *          references or, in case an error occurs during reference resolution,
+     * @return a {@link CompletionStage} that contains, as a result, a new categoryDraft instance with resolved
```

Comment:
I don't think you need commas here - it's essential info: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_with_nonessential_elements.html

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/df1f7503263b9ee67680d464a46fbf391caa3df1

Reply:
but this is in several places, like here: https://github.com/commercetools/commercetools-sync-java/pull/319/files/e328c7f92531b07561a117ba20833fdd52c154e2#diff-365abd2d37edf8522541a0b8b126963fR27 and other places

Reply:
@ahmetoz stale link.

New link: https://github.com/commercetools/commercetools-sync-java/pull/319/files/e328c7f92531b07561a117ba20833fdd52c154e2#diff-365abd2d37edf8522541a0b8b126963fR27